src/pages/_app.tsx

```diff
@@ -1,16 +1,22 @@
 import type { AppProps } from "next/app";
-import { LanguageProvider } from "translations/context/LanguageContext";
 import { Footer, Navigation } from "components/ui";
 import "styles/globals.css";
 import "styles/variables.css";
+import { NextIntlProvider } from "next-intl";
 
 function App({ Component, pageProps }: AppProps) {
+  const { locale, translations } = pageProps.localization;
+
   return (
-    <LanguageProvider localization={pageProps.localization}>
+    <NextIntlProvider
+      messages={translations}
+      locale={locale}
+      defaultTranslationValues={{ b: (children) => <b>{children}</b> }}
```

Comment:
if we're using `t.rich` with some custom tags - we need to provide the implementation for them, we can do that in place where `t.rich` is used or we can provide some global defaults